diesel_cli/src/infer_schema_internals/information_schema.rs

```diff
@@ -124,15 +125,15 @@ where
     use self::information_schema::columns::dsl::*;
 
     let schema_name = match table.schema {
-        Some(ref name) => name.clone(),
-        None => Conn::Backend::default_schema(conn)?,
+        Some(ref name) => Cow::Borrowed(name),
+        None => Cow::Owned(Conn::Backend::default_schema(conn)?),
     };
 
     let type_column = Conn::Backend::type_column();
     columns
         .select((column_name, type_column, is_nullable))
         .filter(table_name.eq(&table.name))
-        .filter(table_schema.eq(schema_name))
+        .filter(table_schema.eq(schema_name.as_ref()))
```

Comment:
`Cow` already implements `AsExpression` so no need to call `.as_ref()` here explicitly.
```suggestion
        .filter(table_schema.eq(schema_name))
```